app/src/main/java/com/example/pygmyhippo/MainActivity.java

```diff
@@ -207,7 +207,7 @@ public void setupNavControllerAdmin(BottomNavigationView navView, Bundle bundle)
      * @param flags - Flags to indicate query status/set how to process query result.
      */
     @Override
-    public void OnComplete(@NonNull ArrayList<Account> docs, int queryID, int flags) {
+    public void OnCompleteDB(@NonNull ArrayList<Account> docs, int queryID, int flags) {
         switch (queryID) {
```

Comment:
Will changing the names of the listeners affect the code anywhere else?

Reply:
I refactored the code so it should have changed everywhere else too

Reply:
Alright. I think it would be best if we all review this change because I know it might affect my PR, we would have to merge this one first